src/main/java/org/sonar/plugins/stash/client/StashClient.java

```diff
@@ -97,6 +100,11 @@ public String getLogin() {
     return credentials.getLogin();
   }
 
+

```

Comment:
This PR seems to use two blank lines between functions, while the existing code only uses one.